Automattic/wp-calypso
# Stats: Add new feature flag for feedback

Related to: https://github.com/Automattic/red-team/issues/155

## Proposed Changes

Adds the "stats/user-feedback" feature flag and an empty placeholder component for the new feedback card. Flag is enabled in development only and the placeholder component is present to confirm the flag is working correctly. We'll build that out in subsequent PRs.

## Why are these changes being made?


Part of the User Feedback project.

pejTkB-1Ce-p2

## Testing Instructions



* Spin up Live branch.
* Visit the Traffic page.
* Confirm the placeholder card is not visible at the bottom of the page (just above the colophon).
* Enable feature flag: `?flags=stats/user-feedback`
* Confirm the placeholder card is now visible.

<img width="722" alt="SCR-20240827-okkr" src="https://github.com/user-attachments/assets/55147e04-3736-460d-95b5-040d072ef687">

## Pre-merge Checklist
